src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -527,6 +527,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_win_init(MPIR_Win * win)
     MPIR_ERR_CHKANDSTMT(window_instance >= MPIDI_Global.max_huge_rmas, mpi_errno,
                         MPI_ERR_OTHER, goto fn_fail, "**ofid_mr_reg");
 
+    assert(MPIDI_Global.max_rma_key_bits > MPIDI_Global.context_shift);
```

Comment:
Should this not be `MPIR_Assert` instead of the regular `assert` call?

Reply:
It should.  But it'll anyway disappear in Yanfei's fixup commit that's coming soon.  The broader issue is that we have been using `assert` directly in the code in a number of places.  We should change all of them to `MPIR_Assert`.